stable/prisma/templates/deployment.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ template "prisma.fullname" . }}
+  labels:
+    app: {{ template "prisma.name" . }}
+    chart: {{ template "prisma.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: {{ template "prisma.name" . }}
+      release: {{ .Release.Name }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: {{ template "prisma.name" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: prisma
+              containerPort: 4466
+              protocol: TCP
+          env:
+            - name: CONFIG
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ template "prisma.fullname" . }}
+                  key: config
+            {{- if .Values.secret }}
+            - name: PRISMA_API_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "prisma.fullname" . }}
+                  key: api-secret
+            {{- end}}
+            - name: PRISMA_CONFIG_PATH
+              value: "/app/config.yml"
+            - name: PRISMA_DB_CONNECTOR
+              value: {{ .Values.database.connector | quote }}
+            - name: PRISMA_DB_HOST
+              value: {{ template "prisma.database.host" . }}
+            - name: PRISMA_DB_PORT
+              value: {{ template "prisma.database.port" . }}
+            - name: PRISMA_DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "prisma.fullname" . }}
+                  key: db-user
+            - name: PRISMA_DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "prisma.fullname" . }}
```

Comment:
Should you not use the password generated in Postgres Secret if ".Values.postgresql.enabled" is true?

Reply:
Good idea! Done